Core/Utilities/MySQL.py

```diff
@@ -1488,7 +1477,7 @@ def insertFields( self, tableName, inFields = None, inValues = None, conn = None
                          ( table, inFieldString, inValueString ), conn, debug = True )
 
 
-  def executeStoredProcedure( self, packageName, parameters, outputIds, output = True, array = None, conn = False ):
+  def executeStoredProcedure( self, packageName, parameters, outputIds ):
```

Comment:
Not sure of that one. I put this useless params to have the same interface as the Oracle interface